resources/views/community/components/request-list/meta-panel.blade.php

```diff
@@ -0,0 +1,69 @@
+<script>
+/**
+ * Updates a query parameter in the current URL and navigates to the new URL.
+ *
+ * @param {string} paramName - The name of the query parameter to update.
+ * @param {string} newQueryParamValue - The new value for the query parameter.
+ */
+ function updateUrlParameter(paramName, newQueryParamValue) {
```

Comment:
Unfortunately, this JS is all untestable. Node.js and JSDOM support only an extremely thin implementation of `URLSearchParams`.